### ISSUE TITLE
triggers-3.sql REMARK

### ISSUE THREAD

**Issue by BertHG** [vertonghenb/DBII-Workshops#29]
In the solution of [triggers-3.sql](https://github.com/vertonghenb/DBII-Workshops/blob/master/workshops/cursors/solutions/cursors-1.sql) there is mentioned:

> deleted virtual table contains copies of updated or inserted rows

Isn't it only the updated and deleted rows that are included in the Deleted Virtual Table ?

**Comment by vertonghenb** [comment]
@BertHG Correct! The virtual table of the `Deleted` rows only contains Updated and/or Deleted rows, inserted rows are never present there. Do you mind creating a PR for this?
